static/js/app.js

```diff
@@ -235,12 +235,35 @@ function addEventListeners() {
     });
   }
 
+
+function moveDownMainContainer(setMenuLinksHeightVal) {
+  const mainWrapper = document.querySelector("main");
+  const moveDownBy = parseInt(setMenuLinksHeightVal);
```

Comment:
I don't think it matters here, but it's generally good practice to always specify the radix (usually 10) when using `parseInt`, because otherwise it will use a radix of 16 when the given string happens to start with `0x`, so that's a good habit to get into: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt

```suggestion
  const moveDownBy = parseInt(setMenuLinksHeightVal, 10);
```

Reply:
Done!

Reply:
Never heard of that before, good to know.